ios/Podfile

```diff
@@ -8,6 +8,9 @@ install! 'cocoapods',
 platform :ios, '10.0'
 inhibit_all_warnings!
 
+# Force all Expo modules to be compiled from source
+ENV['EXPO_USE_SOURCES'] = '1'
```

Comment:
As per online discussion maybe it makes sense to move this to direnv configuration